salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -3787,6 +3762,10 @@ spec:
         #- name: cni-net-dir
         #  hostPath:
         #    path: /etc/cni/net.d
+        # Used to access CNI logs.
+        - name: cni-log-dir
+          hostPath:
+            path: /var/log/calico/cn
```

Comment:
This seems odd. Not `cni`?

Reply:
Also, should we create this on the host (using Salt)?

Reply:
Ouh right typo on my side, good catch

Hmm I don't know :thinking: I need to test

Reply:
We do not need to create the directory it's created automatically